protocols/1291-griffith-university-part3/cell_culture_part3.ot2.py

```diff
@@ -35,8 +41,7 @@
 
 tip_count = 0
 
-dests = [col[well_index]
-         for col in destination.cols()
+dests = [[col[well_index] for col in destination.cols('4', to='21')]
```

Comment:
Maybe we should make this a custom variable too? Let's talk on Monday cus I think you did other edits for these people before.